lib/libspl/include/assert.h

```diff
@@ -107,17 +107,26 @@ do {									\
 	const uintptr_t __right = (uintptr_t)(RIGHT);			\
 	if (!(__left OP __right))					\
 		libspl_assertf(__FILE__, __FUNCTION__, __LINE__,	\
-		    "%s %s %s (0x%llx %s 0x%llx)", #LEFT, #OP, #RIGHT,	\
-		    (u_longlong_t)__left, #OP, (u_longlong_t)__right);	\
+		    "%s %s %s (%px %s %px)", #LEFT, #OP, #RIGHT,	\
+		    (void *)__left, #OP, (void *)__right);		\
```

Comment:
Here %p makes sense, but why %px, considering it is user-space, where I am not sure %px is a thing on any platform.

Reply:
Patch got mangled in transfer between trees, will regenerate.